public/request/forum-topic-comment/preview.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use App\Site\Enums\Permissions;
+use App\Support\Shortcode\Shortcode;
+use Illuminate\Support\Arr;
+use Illuminate\Support\Facades\Blade;
+use Illuminate\Support\Facades\Validator;
+
+if (!authenticateFromCookie($user, $permissions, $userDetails, Permissions::Registered)) {
+    return back()->withErrors(__('legacy.error.permissions'));
+}
+
+$input = Validator::validate(Arr::wrap(request()->post()), [
+    'body' => 'required|string|max:60000',
+]);
+
+$parsedPostContent = Shortcode::render($input['body']);
+
+$postPreviewHTML = '';
```

Comment:
Minor nit: While it's not defined in the PSR coding standard guidelines, acronyms are usually treated like this: `$postPreviewHtml`.
We have a few areas in this codebase where this should be cleaned up.